### PR TITLE
Renamed UI README to BUILD.rst. Added more content to README.

### PR DESCRIPTION
**Pull request by bdmogal**

### CODE REVIEW

File: cdap-ui/README.rst [hunk 1]
```diff
@@ -1,99 +1,120 @@
...
+6. A new ``ui`` directory should be created
+7. Using the command prompt, start the CDAP SDK
 
-If you are working on common components shared between all the apps (for instance, the Header)
-then you need to build an additional ``common`` library that is used across all:
+::
 
-- ``$ webpack --config webpack.config.common.js -d ## build version``
-- ``$ webpack --config webpack.config.common.js --watch -d ## watch version``
+  $ cd $(CDAP_HOME}
```
Comment (wolf31o2) [inline cdap-ui/README.rst]: ${

File: cdap-ui/README.rst [hunk 2]
```diff
@@ -1,99 +1,120 @@
...
+
+Release Notes
+=============
+
+New Features
+------------
+* HYDRATOR-163 - Add Placeholders to input boxes in node configuration
+* WRANGLER-77 - Added a dropdown on each column to provide click-through experience for directives in Data Preparation
+* WRANGLER-49 - Added click-through experience for split column directive in Data Preparation
+* WRANGLER-54 - Added click-through experience for filling null or empty cells in Data Preparation
```
Comment (wolf31o2) [inline cdap-ui/README.rst]: Should these be links to the issues?

**Review by wolf31o2 — CHANGES_REQUESTED** [review]
One minor change and a question

File: cdap-ui/BUILD.rst [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+===============
+
+Prerequisites
+-------------
+- NodeJS Version: 4.5.0 or higher
+
+  CDAP UI requires a minimum NodeJS version of 4.5.0.
+  You could either download from the nodejs.org website or use a version manager.
+
+  - `v4.5.0 Download <https://nodejs.org/download/release/v4.5.0/>`__
```
Comment (elfenheart) [inline cdap-ui/BUILD.rst:18]: let's not suggest this. Downloading from this can cause folder permission issue. Let's just suggest `NVM`

Comment (bdmogal) [reply cdap-ui/BUILD.rst:18]: I just renamed this file. Can we fix it in a follow up? Otherwise give me the instructions, and I'm happy to do it

**Comment by bdmogal** [comment]
@awholegunch has reviewed this via #8406. Merging.

**Pull request merged**